src/main/java/core/basesyntax/strategy/handler/IncrementHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.OperationHandler;
+
+public abstract class IncrementHandler implements OperationHandler {
+    @Override
+    public void apply(Fruit fruit, int quantity) {
```

Comment:
The apply method should not directly manipulate the Storage. Instead, consider using a service that abstracts the storage manipulation logic, which would make the code more maintainable and adhere to the Single Responsibility Principle from SOLID principles.